lib/friendly_shipping/services/usps/serialize_rate_request.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+require 'friendly_shipping/services/usps/machinable_package'
+
+module FriendlyShipping
+  module Services
+    class Usps
+      class SerializeRateRequest
```

Comment:
I think we need a spec for this class. One that tests the usual First Class or Priority shipping methods as well as a Regional Rate shipping method, checking that Service=PRIORITY COMMERCIAL and Container=REGIONALRATEBOXA (or whatever is used).

Reply:
Yeah! I forgot pulling the spec for this one, thank you for reminding me. Spec is now added. 

You will notice that we don't automatically quote commercial now if a flat rate box is used, but that that has to be decided from the calling code.